diff --git a/devLib/Makefile b/devLib/Makefile
index d62b532..4063ea3 100644
--- a/devLib/Makefile
+++ b/devLib/Makefile
@@ -24,6 +24,7 @@
 VERSION=$(shell cat ../VERSION)
 DESTDIR=/usr
 PREFIX=/local
+PWD=`pwd`
 
 STATIC=libwiringPiDev.a
 DYNAMIC=libwiringPiDev.so.$(VERSION)
@@ -31,7 +32,7 @@ DYNAMIC=libwiringPiDev.so.$(VERSION)
 #DEBUG	= -g -O0
 DEBUG	= -O2
 CC	= gcc
-INCLUDE	= -I.
+INCLUDE	= -I. -I$(PWD)/../wiringPi
 DEFS	= -D_GNU_SOURCE
 CFLAGS	= $(DEBUG) $(DEFS) -Wformat=2 -Wall -Winline $(INCLUDE) -pipe -fPIC
 
